homeassistant/components/nextcloud/coordinator.py

```diff
@@ -0,0 +1,75 @@
+"""Data update coordinator for the Nextcloud integration."""
+
+import logging
+from typing import Any
+
+from nextcloudmonitor import NextcloudMonitor, NextcloudMonitorError
+
+from homeassistant.const import CONF_SCAN_INTERVAL, CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DEFAULT_SCAN_INTERVAL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class NextcloudDataUpdateCoordinator(DataUpdateCoordinator["dict[str, Any]"]):
```

Comment:
No need for the quotes here.
```suggestion
class NextcloudDataUpdateCoordinator(DataUpdateCoordinator[dict[str, Any]]):
```